### DISCUSSION TITLE
Can't access main github account

### DISCUSSION THREAD

**Discussion by ooglykraken2** [community/community#118761] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I've been trying to access my original github account and had to create a new one to post. I've tried resetting my password but i've not received any emails to do so, this includes within my spam. If I could get help getting back into my account I'd really appreciate it.

**Comment by eliz-amst** [comment]
Double-Check Email Address.Ensure that you're entering the correct email address associated with your original GitHub account when attempting to reset your password. It's possible that a typo or mistake in the email address could prevent you from receiving the password reset email. Check Spam and Junk Folder, it's worth double-checking to ensure that the password reset email didn't end up there. Sometimes, email filters can be overly aggressive and mistakenly classify legitimate emails as spam. Contact GitHub Support. You can submit a support request through the GitHub Help form (https://support.github.com/contact).

**Comment by ebndev** [comment]
Hi @ooglykraken2,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.
